lib/y2_installbase.pm

```diff
@@ -144,10 +144,18 @@ Performs steps needed to go from the "pattern selection" screen to
 =cut
 sub go_to_search_packages {
     my ($self) = @_;
-    send_key 'alt-d';    # details button
-    assert_screen 'packages-manager-detail';
-    assert_and_click 'packages-search-tab';
-    assert_and_click 'packages-search-field-selected';
+    if (check_var('VIDEOMODE', 'text')) {
```

Comment:
If we are also covering ncurses, I would prefer to handle both QT and ncurses with send_key instead of assert_and_click.

Reply:
+1 . But we should keep in mind if any previous changes went for needle checks and clicks to potentially stabilize which is a common approach. We should prevent to go back-and-forth :)

Reply:
I think that the problem send_key vs assert_and_click is actually that people then to not use enough assert_screen to sync between send_keys and race condition happens.

Reply:
that's a different problem. I just know that e.g. the YaST installer dynamically can assign hotkeys so IIRC there are changes already in this regard for the installer code